python-sdk/example_dags/example_load_file.py

```diff
@@ -253,4 +253,17 @@
     )
     # [END load_file_example_19]
 
+    # [START load_file_example_20]
```

Comment:
since it has tag `# [START load_file_example_20]` you should add this in the documentation, WDYT?

Reply:
Linked it to load example for SFTP